src/main/java/net/revelc/code/formatter/AbstractCacheableFormatter.java

```diff
@@ -44,7 +44,14 @@ public Result formatFile(File file, LineEnding ending, boolean dryRun) {
         try {
             this.log.debug("Processing file: " + file + " with line ending: " + ending);
             String code = FileUtils.fileRead(file, this.encoding.name());
-            String formattedCode = doFormat(code, ending);
+            LineEnding formatterLineEnding = ending;
+            // ISSUE-307: if the line ending is set as KEEP we have to determine the current line ending of the file
```

Comment:
No need to reference the issue (the comment is sufficient, and there's no guarantee the project will always use the same issue tracker), but if you do reference it, it's probably better to use GitHub issue syntax rather than JIRA syntax (which, we're obviously not using):

```suggestion
            // if the line ending is set as KEEP we have to determine the current line ending of the file
```

or

```suggestion
            // Issue #307: if the line ending is set as KEEP we have to determine the current line ending of the file
```

Reply:
Yeah, you're right. I'm using Jira too often...
Removed the issue reference